freud/diffraction.pyx

```diff
@@ -0,0 +1,316 @@
+# Copyright (c) 2010-2020 The Regents of the University of Michigan
+# This file is from the freud project, released under the BSD 3-Clause License.
+
+R"""
+The :class:`freud.diffraction` module provides functions for computing the
+diffraction pattern of particles in periodic systems.
```

Comment:
```suggestion
diffraction pattern of particles in systems with long range order.
```

Diffraction can be used to detect aperiodic, but highly ordered structures like quasicrystals as well.